src/modules/logicModule/LM_PC_PaymentRouter_v1.sol

```diff
@@ -127,4 +155,32 @@ contract LM_PC_PaymentRouter_v1 is
             IERC20PaymentClientBase_v1(address(this))
         );
     }
+
+    /// @dev	Creates the flags and data for a payment order.
+    /// @param  start_ The start date of the streaming period (optional).
+    /// @param  cliff_ The duration of the cliff period.
+    /// @param  end_ The ending of the streaming period.
+    function _assemblePaymentConfig(uint start_, uint cliff_, uint end_)
```

Comment:
You use a function to setup the PaymentConfig here but in the Recurring Payment you dont
Do we want to generalize the assembling into an abstract function maybe?
Just so we agree on a style